modules/accounts/accounts.js

```diff
@@ -39,8 +40,8 @@ class Accounts{
     async listAccounts() {  
         Logger.verbose(`Command : tezster list-accounts`);
         if(Object.keys(config.accounts).length > 0) {
-            for(var i in config.accounts) {
-                Logger.info(config.accounts[i].label + ' - ' + config.accounts[i].pkh + '(' + config.accounts[i].identity + ')');
+            for(var objIndex in config.accounts) {
```

Comment:
use 'let' and 
variable can be named as 'accountIndex'.
if you use forEach, you don't have to deal with indexes.